x/concentrated-liquidity/swaps.go

```diff
@@ -486,8 +497,8 @@ func (k Keeper) computeInAmtGivenOut(
 	// initialize swap state with the following parameters:
 	// as we iterate through the following for loop, this swap state will get updated after each required iteration
 	swapState := SwapState{
-		amountSpecifiedRemaining: desiredTokenOut.Amount.ToDec(), // tokenOut
-		amountCalculated:         sdk.ZeroDec(),                  // tokenIn
+		amountSpecifiedRemaining: tokenAmountOutSpecified, // tokenOut
```

Comment:
ah I now see Adam/Matt's point about inconsistency

Reply:
Replied in the thread above